frontend/src/components/constants.js

```diff
@@ -17,3 +17,6 @@ export const QUERY_CONDITIONS = {
   [WITHIN]: 'win',
   [IS]: 'is',
 };
+
+export const DATE_FORMAT = 'MM/DD/YYYY';
+export const DATETIME_DATE_FORMAT = 'YYYY/MM/DD';
```

Comment:
This is already defined as a constant in the root of the frontend as `DATE_FORMAT`. Another task for another day, I think it might be time to discuss if we want to keep these constant files tucked away in the project or just have a single constants file in the root of the frontend folder so we don't have duplicate constants defined.